sdk/communication/azure-communication-messages/src/main/resources/META-INF/azure-communication-messages_apiview_properties.json

```diff
@@ -17,7 +17,6 @@
     "com.azure.communication.messages.NotificationMessagesClient.send": "Azure.Communication.MessagesService.NotificationMessagesClient.send",
     "com.azure.communication.messages.NotificationMessagesClient.sendWithResponse": "Azure.Communication.MessagesService.NotificationMessagesClient.send",
     "com.azure.communication.messages.NotificationMessagesClientBuilder": "Azure.Communication.MessagesService.NotificationMessagesClient",
-    "com.azure.communication.messages.implementation.models.MessageDataStream": "Azure.Communication.MessagesService.MessageDataStream",
```

Comment:
One less class?

However, I cannot find which change in typespec source causes this.

I verified that this `MessageDataStream` class is not used by code or test.